contracts/0.8.25/vaults/VaultHub.sol

```diff
@@ -1499,6 +1502,7 @@ contract VaultHub is PausableUntilWithRoles {
 
```

Comment:
Looks like it's worth to extract `updateVaultFees` to OperatorGrid to group all business-related logic together.